kt/src/main/kotlin/org/example/Collections.kt

```diff
@@ -0,0 +1,68 @@
+package org.example
+
+import java.util.Collections
+import java.util.concurrent.ConcurrentHashMap
+
+fun example14() {
+//    val list = mutableListOf<Int>()
```

Comment:
Remove comment?

Reply:
nope, it's intended to discuss different constructors and possible caveats